dll/ntdll/ldr/ldrpe.c

```diff
@@ -964,6 +949,10 @@ LdrpLoadImportModule(IN PWSTR DllPath OPTIONAL,
                        &(*DataTableEntry)->InInitializationOrderLinks);
     }
 
+done:
+    /* Do we have a redirect string? */
```

Comment:
Just drop the comment 🙂

Reply:
right. I had a check to see if there is something in the string first but it wasn't needed and I forgot to remove the comment when I removed the check.